src/site/content/en/notifications/codelab-notifications-service-worker/index.md

```diff
@@ -0,0 +1,192 @@
+
+## Register the service worker
+
+In this step, you'll write code that runs
+when the user clicks **Register service worker** in the app UI.
+This code will register `public/service-worker.js` as a service worker.
+
+1.  In the embedded Glitch editor, open `public/index.js`.
+    Replace the `registerServiceWorker` function with the following code:
+
+    ```js
+    // Use the Service Worker API to register a service worker.
+    async function registerServiceWorker() {
+      await navigator.serviceWorker.register('./service-worker.js')
+      updateUI();
+    }
+    ```
+
+    Note that `registerServiceWorker` uses the [`async function`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function) declaration to make handling promises more convenient. This lets you `await` the resolved value of a `Promise`. For example, the function above awaits the outcome of registering a service worker before updating the UI. See [`await` on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await) for more information.
+
+1.  Now that the user can register a service worker, you can get a reference to the service worker registration object. In `public/index.js`, replace the `getRegistration` function with the following code:
+
+    ```js
+    // Get the current service worker registration.
+    function getRegistration() {
+      return navigator.serviceWorker.getRegistration();
+    }
+    ```
+
+    The function above uses the
+    [Service Worker API](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API)
+    to get the current service worker registration, if it exists.
+    It makes getting a reference to the service worker registration a bit more convenient.
+
+*   To complete the service worker registration functionality, add code to unregister the service worker. Replace the `unRegisterServiceWorker` function with the following code:
+
+    ```js
+    // Unregister a service worker, then update the UI.
+    async function unRegisterServiceWorker() {
+      // Get a reference to the service worker registration.
+      let registration = await getRegistration();
+      // Await the outcome of the unregistration attempt
+      // so that the UI update is not superceded by a 
+      // returning Promise.
+      await registration.unregister();
+      updateUI();
+    }
+    ```
+
+In the tab where you're viewing the live app, reload the page. The **Register service worker** and **Unregister service worker** buttons should now be working.
+
+## Send notifications to the service worker
+
+In this step, you'll write code that will run when the user clicks **Send a notification** in the app UI. This code will create a notification, check that a service worker is registered, and then send the notification to the service worker using its `postMessage` method.
+
+In the embedded Glitch editor, open `public/index.js` and
```

Comment:
Since this is a one-item list, I just made it a paragraph, which leaves more horizontal space for this and the following content.